src/iexamine.cpp

```diff
@@ -7288,8 +7288,7 @@ void iexamine::workbench_internal( Character &you, const tripoint &examp,
                     break;
                 }
                 const recipe &rec = selected_craft->get_making();
-                const inventory &inv = you.crafting_inventory();
-                if( !you.has_recipe( &rec, inv, you.get_crafting_group() ) ) {
+                if( !you.has_recipe( &rec ) ) {
```

Comment:
Same theory as above. Player is able to continue craft without the skills required.

Reply:
Yup, this is true too.

Reply:
Can do on master d30757ea5f too

Reply:
current status on my local branch: fixed, but there is no feedback, so needs further work

Reply:
Decided against fixing it here.